lms/templates/courseware/courseware-chromeless.html

```diff
@@ -71,7 +71,9 @@
 
 <div class="course-wrapper">
   <section class="course-content" id="course-content">
-    ${HTML(fragment.body_html())}
```

Comment:
I don't know if the "chromeless" version of the courseware is supposed to be accessible, but if it is, I believe this change is necessary.